engines/wasmtime/wasmtime.go

```diff
@@ -87,9 +107,21 @@ func (e *engine) Name() string {
 
 // New implements the same method as documented on wapc.Engine.
 func (e *engine) New(_ context.Context, host wapc.HostCallHandler, guest []byte, config *wapc.ModuleConfig) (mod wapc.Module, err error) {
-	engine := wasmtime.NewEngine()
+	cfg := wasmtime.NewConfig()
+	if e.isDebug {
+		cfg.SetDebugInfo(true)
+	}
+	if e.useMetrics {
+		cfg.SetConsumeFuel(true)
+	}
+	engine := wasmtime.NewEngineWithConfig(cfg)
 	store := wasmtime.NewStore(engine)
-
+	if e.useMetrics {
+		err := store.AddFuel(e.maxGas)
```

Comment:
maybe settle on one noun, seems wasmer uses both fuel and points. I think we shouldn't add a third word, gas? Also, if you pass the config instead then you don't have to decide what to name it.

Reply:
Agreed.  My bad.  Started with trying to normalize all to metrics only to reverse course.  I will make thing internally consistent.

Reply:
wasmtime actually uses the term gas in their docs for their metering support.  That was where I got it from.  I was thinking that if we use the native terms here it will indicate to the caller that there are subtle differences that should not be ignored.

Reply:
maybe put this in a comment to prevent confusion?
```suggestion
		// wasmtime uses the word gas in docs even though their API calls it fuel.
		err := store.AddFuel(e.maxGas)
```

Reply:
My bad.  We use the term gas in the blockchain world which is where gas came from.  Changing to fuel.  Sorry for the confusion